ledger-tool/src/main.rs

```diff
@@ -2141,32 +2171,6 @@ fn main() {
                         debug!("Account removed: {address}");
                     }
 
-                    for address in feature_gates_to_deactivate {
```

Comment:
do we need to retain this code under a new
`if !child_bank_required {`
Otherwise, this code no longer runs `if !child_bank_required`

I do not know when `child_bank_required` is true or false.

Reply:
https://github.com/anza-xyz/agave/pull/4465/files#r1916860908